gui/gui/src/ts/components/pages/HomePage.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import * as qs from "qs";
+import {NodeSearchBox} from "components/search/NodeSearchBox";
+import {HomePageQuery} from "api/queries/types/HomePageQuery";
+import {useQuery} from "@apollo/react-hooks";
+import * as HomePageQueryDocument from "api/queries/HomePageQuery.graphql";
+import {Frame} from "components/frame/Frame";
+
+import {
+  Grid,
+  Container,
+  Typography,
+  makeStyles,
+  createStyles,
+  Button,
+} from "@material-ui/core";
+
+import {useHistory} from "react-router-dom";
+
+import {Hrefs} from "Hrefs";
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    container: {
+      paddingTop: theme.spacing(5),
+    },
+    title: {
+      fontFamily: "Hiragino Maru Gothic Pro",
+    },
+    primaryText: {
+      color: theme.palette.primary.main,
+    },
+  })
+);
+
+export const HomePage: React.FunctionComponent<{}> = () => {
```

Comment:
Nit: don't need <{}> if you don't take props, it's implied. (TypeScript has default type parameters, in this case Props = {}).

Reply:
Changed